src/selection/SelectionUtils.js

```diff
@@ -50,7 +50,8 @@ export const rangeToSelection = (range, startNode) => {
   const quote = range.toString();
   const start = rangeBefore.toString().length;
 
-  return new Selection({ 
+  return new Selection(
+    user, { 
```

Comment:
A change to client-core is necessary as the interface of `Selection` has now changed.

Reply:
This means we need to fork the client-core lib as well and maintain it.  Shall we create a new ticket?

Reply:
Yep, I've already forked the client-core lib as well https://github.com/Pathlight/recogito-client-core.